components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { Radio, RadioGroup } from "./index"
+
+storiesOf("Radio", module)
```

Comment:
Add tests for testing the behavior on changing selections.

Reply:
Done